src/app/common/models/Table/Table-config.ts

```diff
@@ -1,9 +1,10 @@
 import { TableHeaderConfig, Paginator } from './index';
-import { EditMark } from '../edit-mark';
+import { EditMark } from '../mark/edit-mark';
+import { TNullable } from '../useful/tnullable';
 
 export interface ITableConfig<T> {
   headers: Array<TableHeaderConfig>;
   body: Array<T>;
-  paginator?: Paginator;
-  editCell?: EditMark;
+  paginator?: TNullable<Paginator>;
+  editCell?: TNullable<EditMark>;
```

Comment:
so this field could be EditMark or null or undefined. Sure need both null and undefined